etc/eva_project_validation.yaml

```diff
@@ -0,0 +1,119 @@
+#Configuration for defining the cerberus validation.
```

Comment:
Just for my reference, I am guessing this was constructed based on the JSON sent to DSP on Feb 4, 2020 (https://www.ebi.ac.uk/panda/jira/browse/EVA-1786). Is that right?

Reply:
No this was constructed from reading the validation done in the perl script in the [metadata repo](https://github.com/EBIvariation/metadata/blob/master/submission_to_xml/eva_submission_to_xml_V1.1.0.pl) 
I will look at the JSON schema file to see if additional validation could apply.